exercise_07/isTypeOf.js

```diff
@@ -6,3 +6,15 @@
  * input: boolean (type)
  * output:
  */
+
+/**
+ *
+ * @param {Array,Number,String,Boolean} value
+ * @param {String} type
+ * @returns {boolean} True if value matches type
+ */
+function isTypeOf(value, type) {
+  if (type === Array.isArray() && Array.isArray(value)) return true;
```

Comment:
Wrong... many solutions with this error. Have you tested it? 🤔

FIX: needs work